pandas_gbq/auth.py

```diff
@@ -43,10 +49,11 @@ def get_credentials(
 
     credentials, default_project_id = pydata_google_auth.default(
         SCOPES,
-        client_id=CLIENT_ID,
-        client_secret=CLIENT_SECRET,
+        client_id=client_id,
```

Comment:
We need to keep our default client ID and secret.

Note: if `auth_local_webserver = False`, then ideally we provide a different default client ID & secret & a default redirect URI, but we can/should do that in a follow-up PR.